blaze/expr/tests/test_table.py

```diff
@@ -201,7 +189,7 @@ def test_path_issue():
     t2 = transform(t, sizes=t.result.map(lambda x: (x - MIN)*10/(MAX - MIN),
                                          schema='float64', name='size'))
 
-    assert t2.sizes in t2.children
+    assert builtins.any(t2.sizes.isidentical(node) for node in t2.children)
```

Comment:
hm nevermind, we often want to check a specific list of things, not the actual expression